patches/server/1065-Create-Limited-set-for-Entity.patch

```diff
@@ -0,0 +1,71 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Intybyte <vaan.testwork@gmail.com>
+Date: Mon, 30 Sep 2024 17:16:36 +0200
+Subject: [PATCH] Create Limited set for Entity
+
+
+diff --git a/src/main/java/io/papermc/paper/util/SizeLimitedSet.java b/src/main/java/io/papermc/paper/util/SizeLimitedSet.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..d6789463dc8d671f3dc1f011e1c81ae94d105a95
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/util/SizeLimitedSet.java
+@@ -0,0 +1,37 @@
++package io.papermc.paper.util;
++
++import org.jspecify.annotations.NullMarked;
++import java.util.Collection;
++import java.util.HashSet;
++
++@NullMarked
++public class SizeLimitedSet extends HashSet<String> {
```

Comment:
I feel the name is too generic for what this does unless you add a generic to it and remove tag reference.